pyproject.toml

```diff
@@ -81,7 +81,7 @@ ignore_missing_imports = true
 requires = [
     "setuptools ~= 44.1.1",
     "wheel ~= 0.36.2",
-    "incremental == 17.5.0",
+    "incremental == 22.10.0",
```

Comment:
The incremental related tests was failing for the release candidate version format...so at first I try updating incremental with the hope that normalization is fixed.

It is not... but maybe is ok to ask for a newer incremental.

Best, would be to not depend on incremental :)

Let me know if this should be reverted or moved to a separate PR.